### PR TITLE
[teleop, esw] polarity flip

### PR DESCRIPTION
**Pull request by cgiger00**
from testing sunday

### CODE REVIEW

File: jetson/odrive_bridge/src/__main__.py [hunk 1]
```diff
@@ -447,9 +447,9 @@ def _requested_state(self, state):
 
     def set_vel(self, axis, vel):
```
Comment (qhdwight) [inline jetson/odrive_bridge/src/__main__.py:448]: Looks good

File: jetson/teleop/src/__main__.py [hunk 2]
```diff
@@ -133,8 +133,9 @@ def teleop_drive_callback(self, channel, msg):
                 vel_right /= abs(vel_right)
 
         command = DriveVelCmd()
```
Comment (qhdwight) [inline jetson/teleop/src/__main__.py:135]: Would it make more sense to invert it higher up? E.g. on line 111 in teleop:
`linear = deadzone(-input.forward_back, 0.05) * input.dampen`

Comment (cgiger00) [reply jetson/teleop/src/__main__.py:135]: @CameronTressler

Comment (CameronTressler) [reply jetson/teleop/src/__main__.py:135]: Going to push a commit to remove the changes here. Relevant changes for teleop in #1012

**Review by CameronTressler — APPROVED** [review]

**Review by tabiosg — APPROVED** [review]

**Pull request merged**